### PR TITLE
Added README for tutorial.reading_time in functional_samples

### PR DESCRIPTION
**Pull request by mahitoburrito**

### CODE REVIEW

File: functional-samples/tutorial.reading-time/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Tutorial: Run scripts on every page
+
+This sample demonstrates how to _run scripts on every page_ using an extension called _Reading Time_.
```
Comment (oliverdunk) [inline functional-samples/tutorial.reading-time/README.md:3]: In practice, we only support certain documentation pages on developer.chrome.com. Could you update this introduction and the "Running this extension" section to reflect that?

File: functional-samples/tutorial.reading-time/images/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Tutorial: Run scripts on every page
```
Comment (oliverdunk) [inline functional-samples/tutorial.reading-time/images/README.md:1]: We should remove this duplicate in the images directory

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for the PR! I've left some feedback.

File: functional-samples/tutorial.reading-time/README.md (hunk 1 above)
Comment (mahitoburrito) [reply functional-samples/tutorial.reading-time/README.md]: Thanks for catching that! Updated the wording to be more specific.

File: functional-samples/tutorial.reading-time/images/README.md (hunk 2 above)
Comment (mahitoburrito) [reply functional-samples/tutorial.reading-time/images/README.md]: Removed!

**Review by oliverdunk — APPROVED** [review]
This looks good to me! Thanks.

**Pull request merged**